pkg/status/handlers_test.go

```diff
@@ -48,3 +50,71 @@ func TestAliveOK(t *testing.T) {
 	}
 	assert.Equalf(t, "ok", receivedStatus.Status, "Expected %s, got %s", "ok", receivedStatus.Status)
 }
+
+func TestDeepCheckSuccess(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockLogger := NewMockLoggerInterface(ctrl)
+	mockMonitor := NewMockMonitorInterface(ctrl)
+	mockTracer := NewMockTracer(ctrl)
+	mockService := NewMockServiceInterface(ctrl)
+
+	req := httptest.NewRequest(http.MethodGet, "/api/v0/deepcheck", nil)
+	w := httptest.NewRecorder()
+
+	mockTracer.EXPECT().Start(gomock.Any(), gomock.Any()).Times(1).Return(context.TODO(), trace.SpanFromContext(req.Context()))
+	mockService.EXPECT().CheckKratosReady(gomock.Any()).Times(1).Return(true, nil)
+	mockService.EXPECT().CheckHydraReady(gomock.Any()).Times(1).Return(true, nil)
+
+	mux := chi.NewMux()
+	NewAPI(mockService, mockTracer, mockMonitor, mockLogger).RegisterEndpoints(mux)
+
+	mux.ServeHTTP(w, req)
+	res := w.Result()
+	defer res.Body.Close()
+	data, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		t.Fatalf("expected error to be nil got %v", err)
+	}
+	receivedStatus := new(DeepCheckStatus)
+	if err := json.Unmarshal(data, receivedStatus); err != nil {
+		t.Fatalf("expected error to be nil got %v", err)
+	}
+	assert.Equalf(t, "ok", receivedStatus.KratosStatus, "Expected %s, got %s", "ok", receivedStatus.KratosStatus)
+	assert.Equalf(t, "ok", receivedStatus.HydraStatus, "Expected %s, got %s", "ok", receivedStatus.HydraStatus)
```

Comment:
In other test files we use `if` statements to make assertions. Let's be consistent and do it like this here as well

Reply:
ok. Good catch!